examples/hotrod/services/frontend/web_assets/index.html

```diff
@@ -73,7 +73,12 @@ <h4><em>Rides On Demand</em></h4>
   };
   console.log(headers);
   var before = Date.now();
-  $.ajax('/dispatch?customer=' + customer + '&nonse=' + Math.random(), {
+
+  // Use current URI as basepath for ajax requests
+  var pathPrefix = window.location.pathname;
+  pathPrefix = pathPrefix != "/" ? pathPrefix : '';
+
+  $.ajax(pathPrefix + '/dispatch?customer=' + customer + '&nonse=' + Math.random(), {
```

Comment:
the demo still works for me without this change. Why is it necessary? (Perhaps I've missed something?)

Reply:
without this the browser would always talk to `/dispatch` instead of `/something/dispatch`. Any changes to web_assets need to be compiled using `esc`. Perhaps you missed that?

Check this line https://github.com/jaegertracing/jaeger/blob/master/examples/hotrod/services/frontend/gen_assets.go#L1